chartfx-dataset/src/test/java/de/gsi/dataset/spi/DoubleDataSet3DTests.java

```diff
@@ -0,0 +1,177 @@
+package de.gsi.dataset.spi;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Arrays;
+
+import static de.gsi.dataset.DataSet.DIM_X;
+import static de.gsi.dataset.DataSet.DIM_Y;
+import static de.gsi.dataset.DataSet.DIM_Z;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Checks for the DoubleDataSet3D
+ * 
+ * @author Alexander Krimm
+ */
+public class DoubleDataSet3DTests {
+
+    @Test
+    public void testDefaultConstructor() {
+        final DoubleDataSet3D dataset = new DoubleDataSet3D("testdataset");
+        assertEquals("testdataset", dataset.getName());
+        assertEquals(0, dataset.getDataCount());
+        assertEquals(0, dataset.getDataCount(DIM_X));
+        assertEquals(0, dataset.getDataCount(DIM_Y));
+        assertEquals(0, dataset.getDataCount(DIM_Z));
+        assertThrows(IndexOutOfBoundsException.class, () -> dataset.get(DIM_X, 0));
+    }
+
+    @Test
+    public void testZeroSizeConstructors() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5084368)